src/BeeAuth.php

```diff
@@ -71,6 +95,17 @@ public function generateToken()
             ],
         ]);
 
-        return new AuthorizationToken(json_decode($response->getBody()->getContents(), true));
+        $token = new AuthorizationToken(json_decode($response->getBody()->getContents(), true));
+
+        if ($this->cache) {
+            $this->cache->set($this->getCacheKey(), $token->toArray(), ($token->getExpiresIn() - 10));
+        }
+
+        return $token;
+    }
+
+    protected function getCacheKey()
+    {
+        return 'kdg.bee-plugin.bee-auth.token';
```

Comment:
Should we be careful with "kdg" as an acronym? It doesn't matter too much here since it's in the cache but just throwing it out there.

Reply:
It's a good point. I guess I'll just remove `kdg.`.